cluster/operations/windows-worker.yml

```diff
@@ -2,21 +2,23 @@
   path: /releases/-
   value:
     name: concourse-windows-worker
-    version: latest
+    version: ((concourse_windows_worker_version))
+    sha1: ((concourse_windows_worker_sha1))
+    url: https://bosh.io/d/github.com/pivotal-cf-experimental/concourse-windows-worker-release?v=((concourse_windows_worker_version))
 
 - type: replace
   path: /stemcells/-
   value:
     alias: windows
-    os: windows2012R2
+    os: ((windows_worker_os_version))
```

Comment:
This should probably be just `windows_worker_os`

Out of curiosity, what other value did you need to specify here?

Reply:
We (garden windows team) use windows2016 workers in our concourse pipeline